shared/templates/sebool/bash.template

```diff
@@ -6,6 +6,10 @@
 
 {{% if product == "sle15" %}}
 {{{ bash_package_install("policycoreutils") }}}
+{{{ bash_package_install("policycoreutils-python-utils") }}}
```

Comment:
In Ansible at least one package is always installed regardless of the product. But the Ansible install package only for `sle15`. Bash and Ansible should be aligned.

Reply:
Correct, but we should also ensure the installation of semanage library for other distros too, like we do in Ansible (first task).

Reply:
Correct, but we should also ensure the installation of semanage library for other distros too, like we do in Ansible (first task).